### DISCUSSION TITLE
using MySQLDsn or MariaDBDsn

### DISCUSSION THREAD

**Discussion by jakekwak** [pydantic/pydantic#5123] [Question] [answered]
I want to use MySQLDsn or MariaDBDsn.  but the release v1.10.5  is not include this.

When is it available?

**Comment by hramezani** [comment, marked as answer]
`V1.10.x` will get only bug fixes. So, no new feature will be available on it.
It will be available on [V2](https://docs.pydantic.dev/blog/pydantic-v2/).

**Reply by jakekwak** [reply]
Thanks
